Xamarin.Essentials/Calendars/Calendars.tizen.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TCalendar = Tizen.Pims.Calendar;
+using TCalendarViews = Tizen.Pims.Calendar.CalendarViews;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Calendars
+    {
+        static TCalendar.CalendarManager manager = new TCalendar.CalendarManager();
+
+        static Task<IEnumerable<Calendar>> PlatformGetCalendarsAsync()
+        {
+            var calendarList = manager.Database.GetAll(TCalendarViews.Book.Uri, 0, 0);
+            return Task.FromResult<IEnumerable<Calendar>>(ToCalendars(calendarList).ToList());
+        }
+
+        static IEnumerable<Calendar> ToCalendars(TCalendar.CalendarList calendarList)
+        {
+            calendarList.MoveFirst();
+            for (var i = 0; i < calendarList.Count; i++)
+            {
+                yield return ToCalendar(calendarList.GetCurrentRecord());
+                calendarList.MoveNext();
+            }
+        }
+
+        static Calendar ToCalendar(TCalendar.CalendarRecord calendarRecord)
+        {
+            return new Calendar
+            {
+                Id = calendarRecord.Get<int>(TCalendarViews.Book.Id).ToString(),
```

Comment:
ToString uses the Threads current culture. Specify InvariantCulture to not run into problems with numbergroupseperators, i.e. 1234 => 1.234